app/models/hearings/forms/base_hearing_update_form.rb

```diff
@@ -105,6 +105,19 @@ def should_create_or_update_virtual_hearing?
     )
   end
 
+  # This is an existing hearing that is being converted to virtual if:
+  # - There is no virtual hearing
+  # - This hearing exists with a request type that is not virtual ("R")
+  def conversion_to_virtual?
+    hearing.request_type != "R" && !hearing.virtual?
+  end
+
+  # This checks that request_cancelled is being set, that only happens
+  # when converting FROM virtual to the original type
+  def conversion_to_not_virtual?
```

Comment:
i think i prefer this naming better
```suggestion
  def converting_from_virtual?
```

Reply:
I changed this naming, "...from..." is definitely better.